unstructured/partition/strategies.py

```diff
@@ -1,6 +1,7 @@
 from tempfile import SpooledTemporaryFile
 from typing import BinaryIO, Dict, List, Optional, Union, cast
 
+import torch
```

Comment:
Same comment here, I don't think we should import `torch` at the module level in case the `local-inference` extra isn't installed.